requirements/requirements.txt

```diff
@@ -148,22 +148,12 @@ hyperlink==20.0.1
     # via
     #   autobahn
     #   twisted
-idna-ssl==1.1.0
```

Comment:
This is unnecessary in Python3.7+

https://github.com/aio-libs/aiohttp/issues/949
https://github.com/python/cpython/pull/3010